modules/lang-painless/src/test/java/org/elasticsearch/painless/UserFunctionTests.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.painless;
+
+public class UserFunctionTests extends ScriptTestCase {
```

Comment:
I realize this is internal, but would you please add a few more tests of basic scripts just so we can tell if we break something when making changes to phases. I don't want too many either because I realize there's crazy overhead in maintaining them while were still working toward a final model.

Reply:
Added more in `ToXContentTests.java`